README.md

```diff
@@ -105,6 +105,10 @@ To include your `avatar`, You just have to replace `<yourusername>` with your **
 
 For example: <http://github.com/eddiejaoude.png>
 
+### A simple gif version of adding your profile
+
+<https://github.com/KarlGusta/LinkFree-docs/blob/main/Linkfree%20adding%20profile%20gif.gif>
```

Comment:
We need to set this as an image to make it work.
```suggestion
![GIF](https://github.com/KarlGusta/LinkFree-docs/blob/main/Linkfree%20adding%20profile%20gif.gif)
```

Reply:
I'm not sure if I get you @Panquesito7

Reply:
> I'm not sure if I get you @Panquesito7

Oh, I meant to use the image format, which is `![placeholder](image-link)`; otherwise it won't work. Adding the link directly won't show up in an image.
Pradumna's suggestion already fixed this, so all's good now. I hope I get to understand.

Reply:
Thanks for this. I understand it now.